db/organizations.go

```diff
@@ -155,3 +186,43 @@ func (ms *MongoStorage) OrganizationsMembers(address string) ([]User, error) {
 	}
 	return users, nil
 }
+
+// addSubscriptionToOrganization internal method adds the subscription to the organiation with
+// the given address. If an error occurs, it returns the error. This method must
+// be called with the keysLock held.
+func (ms *MongoStorage) AddSubscriptionToOrganization(address string, orgSubscription *OrganizationSubscription) error {
+	if _, err := ms.Plan(orgSubscription.PlanID); err != nil {
+		return ErrInvalidData
+	}
+	ms.keysLock.Lock()
+	defer ms.keysLock.Unlock()
+	// create a context with a timeout
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	// prepare the document to be updated in the database
+	filter := bson.M{"_id": address}
+	updateDoc := bson.M{"$set": bson.M{"subscription": orgSubscription}}
+	// update the organization in the database
+	if _, err := ms.organizations.UpdateOne(ctx, filter, updateDoc); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Implement update subscription method and ensuere that only members
```

Comment:
The function comments in go should start with the name of the function itself: `UpdateOrganizationSubscritpion function implements...`

Reply:
Removed funciont as it was redudant with `SetOrganizationSubscritption`